src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -193,11 +195,14 @@ module.exports = function<T, P, I, TI, C>(
         commitNestedUnmounts(node);
         // After all the children have unmounted, it is now safe to remove the
         // node from the tree.
-        if (parent) {
+        if (parent && !ignoreHostNode) {
           removeChild(parent, node.stateNode);
```

Comment:
Please bear with me as I don't fully understand this method yet.
Can `removeChild()` below run more than once?
If it can, `ignoreHostNode` should be named `ignoreHostNodes`?